apex-squid/src/main/java/org/fundacionjala/enforce/sonarqube/apex/api/ApexKeyword.java

```diff
@@ -92,9 +92,48 @@
     READ_ONLY("ReadOnly"),
     REMOTE_ACTION("RemoteAction"),
     TEST_SETUP("testSetup"),
-    TEST_VISIBLE("TestVisible");
+    TEST_VISIBLE("TestVisible"),
 
     /**
+     * SOQL ANNOTATIONS.
+     */
+    DATABASE("Database"),
+    //Select Notations
+    SELECT("SELECT"),
+    COUNT("COUNT"),
+    //From Notations
+    FROM("FROM"),
+    AS("AS"),
+    USING("USING"),//Pending to implement
+    //With Notations
+    WITH_SOQL("WITH"),
+    //Where Notations
+    WHERE("WHERE"),
+    NOT("NOT"),
```

Comment:
Please review this doc
https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql_select_comparisonoperators.htm

We don't have a NOT keyword but yes NOT IN